src/lint.ts

```diff
@@ -76,7 +76,26 @@ function testDependencies(
         getCurrentDirectory: () => dirPath,
         getNewLine: () => "\n",
     });
-    return `Errors in typescript@${version} for external dependencies:\n${showDiags}`;
+
+    const message = `Errors in typescript@${version} for external dependencies:\n${showDiags}`
+
+    // Add an edge-case for someone needing to `npm install` in react when they first edit a DT module which depends on it - #226
+    const cannotFindDepsDiags = diagnostics.find(d => d.code === 2307 && d.messageText.toString().includes("Cannot find module"))
+    if (cannotFindDepsDiags && cannotFindDepsDiags.file) {
+        const path = cannotFindDepsDiags.file.fileName
+        const typesFolder = basename(dirname(path))
+
+        return `A module look-up failed, the most common case for this error are dependencies which needs to be \`npm installed\` before you can lint.
+        
+Before you debug, first try running: 
+   npm install --prefix types/${typesFolder}
```

Comment:
does this work if you run the `npm test` command from inside `types/` or `types/whatever/`?

Reply:
Good idea, it's now absolute so it doesn't matter